2019-12-06 16:29:43,395 INFO [root] [test_loginihrm.py(test01_login_success:31)] - 登录接口返回的数据为:{'success': True, 'code': 10000, 'message': '操作成功！', 'data': 'aae6d06f-1fb5-418f-ac97-3c325d7e83f9'}
2019-12-06 16:29:43,395 INFO [root] [test_loginihrm.py(test01_login_success:42)] - 保存的登录token和content-type: {'Content-Type': 'application/json', 'Authorization': 'Bearer aae6d06f-1fb5-418f-ac97-3c325d7e83f9'}
2019-12-06 16:29:43,504 INFO [root] [test_loginihrm.py(test02_mobile_is_error:50)] - 登录接口返回的数据为:{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2019-12-06 16:29:43,583 INFO [root] [test_loginihrm.py(test03_password_is_error:59)] - 登录接口返回的数据为:{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2019-12-06 16:29:43,676 INFO [root] [test_loginihrm.py(test04_none_params:68)] - 登录接口返回的数据为:{'success': False, 'code': 99999, 'message': '抱歉，系统繁忙，请稍后重试！', 'data': None}
2019-12-06 16:29:43,770 INFO [root] [test_loginihrm.py(test05_mobile_is_null:77)] - 登录接口返回的数据为:{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2019-12-06 16:29:43,848 INFO [root] [test_loginihrm.py(test06_password_is_null:87)] - 登录接口返回的数据为:{'success': False, 'code': 20001, 'message': '用户名或密码错误', 'data': None}
2019-12-06 16:29:43,957 INFO [root] [test_loginihrm.py(test07_extra_params:96)] - 登录接口返回的数据为:{'success': True, 'code': 10000, 'message': '操作成功！', 'data': 'a66bc9cb-251d-47db-bc2b-0c26ad684b05'}
